src/lib/tls/tls_blocking.cpp

```diff
@@ -6,12 +6,6 @@
 * Botan is released under the Simplified BSD License (see license.txt)
 */
 
-/*
-This API is itself deprecated, so we don't care that it relies on
-other deprecated things
```

Comment:
I found this comment useful. Can we move it down to the use of `TLS::Compat_Callbacks(TLS::Compat_Callbacks::SILENCE_DEPRECATION_WARNING::PLEASE, …`?